### ISSUE TITLE
Maybe move this repo somewhere else?

### ISSUE THREAD

**Issue by mortele** [mortele/variational-monte-carlo-fys4411#4]
@Schoyen Maybe it makes sense to move this repo somewhere else&mdash;is there e.g. a computational-physics github organization? Might be less awkward seeing as I haven't taught the course in more than 6 years, but people still seem to use this code (which also should probably be discouraged at this point 😅).

**Comment by Schoyen** [comment]
Agreed! I'll have a look at some options for moving the repo 👍  The project where this template is relevant is still the same as it was 6 years ago, so the code is – in my opinion – still useful! 😄
